### ISSUE TITLE
Is this abandoned? 

### ISSUE THREAD

**Issue by map588** [radxa-build/radxa-zero3#17]
Would you consider just bolstering another project like [Armbian](https://github.com/armbian/os) with support for your boards?  This is neglectful at this point, and I have to dig around for third party support for anything to work.

I'm sure managing a full distribution of linux is a pain, why not just help make radxa boards work well on _an_ OS.

**Comment by RadxaYuntian** [comment]
We are actually managing a Linux distribution and part of it is online update service. Periodically we [update our package repository](https://github.com/radxa-repo/bullseye/actions) so we don't have to release new system, and existing users don't need to trash their existing system config to flash a new image. The latest kernel for ZERO 3 is built [2 weeks ago](https://github.com/radxa-pkg/linux-rk356x/releases/tag/5.10.160-36) which is hardly abandoned. We are committed in this service model, and are in the progress of [creating system upgrade utility](https://github.com/radxa-pkg/radxa-dist-upgrade/pull/2) for the upcoming Debian 12 release.

In short, we are just doing things more like a traditional distro. Debian doesn't update their ISO every week, neither do us. Forcing users to install new system image for fixes is wrong and unnecessary.

**Comment by map588** [comment]
Apologies.  I'm not familiar enough with this project structure. The latest release in this particular repo is from January.  
I've just been having frustrations getting anything working with this zero 3w, and everything is a rats-nest of repositories between Radxa support for many boards, and rock chip doing whatever they do.  Its a confusing thing to navigate is my point I suppose, and I said that based on bad assumptions. 

I know you all are working hard, and I appreciate your efforts.

**Comment by RadxaYuntian** [comment]
Please make sure you use the image listed in [our official documentation](https://docs.radxa.com/en/zero/zero3/getting-started/download).

Further support questions can be submitted on our forum or Discord server. You can at me on both platforms for additional help.
